mtui/template/testreport.py

```diff
@@ -443,6 +452,8 @@ def _show_yourself_data(self):
                 ("Build checks", self._testreport_url()[:-3] + "build_checks"),
                 ("Testreport", self._testreport_url()),
                 ("Repository", self.repository),
+                ("DashBoard Link", self._get_result_links()[0]),
+                ("OpenQA Link", self._get_result_links()[1]),
```

Comment:
```suggestion
                ("openQA Link", self._get_result_links()[1]),
```

Reply:
I would also lowercase "Link" here, as well as in the line above. If this ends up rendered as link then maybe also drop " Link" completely. The other entries also don't have " Link".